2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1132,28 +1132,31 @@ A seguito dell'implementazione efficace del processo di misurazione:
 - i dati vengono analizzati e i risultati interpretati;
 - gli elementi informativi forniscono informazioni oggettive per poter prendere decisioni concrete.
 
+I risultati sono contenuti nel documento _Piano Di Qualifica_.
+
 === Attività
 
 Il progetto deve implementare le seguenti attività in conformità con le politiche e le procedure applicabili al processo di misurazione definito dal gruppo (parte di questo viene definito nel Piano di Qualifica):
 
 + prepararsi per la misurazione:
 
-    - definire la strategia di misurazione per i processi primari. La strategia scelta si compone di una serie di parametri e metriche che permettono di valutare la qualità dei processi primari:
-        - BAC (Budget at Completion);
-        - AC (Actual Cost);
-        - PV (Planned Value);
-        - EV (Earned Value);
-        - CV (Cost Variance);
-        - SV (Schedule Variance);
-        - CPI (Cost Performance Index);
-        - EAC (Estimated at Completion);
-        - ETC (Estimated to Completion).
+    - definire la strategia di misurazione per i processi primari. La strategia scelta si compone di una serie di parametri e metriche scelte dal gruppo che permettono di valutare la qualità dei processi primari:
```

Comment:
Riformulerei in: "La strategia si compone  ~di una serie di parametri~ di metriche mirate alla valutazione dei processi primari, quali:"